src/lib/transition_frontier/transition_frontier.ml

```diff
@@ -101,6 +101,59 @@ struct
     let create transition_with_hash staged_ledger =
       {transition_with_hash; staged_ledger}
 
+    let build ~logger ~parent ~transition_with_hash =
+      let open Or_error.Let_syntax in
+      let logger = Logger.child logger "transition_frontier_breadcrumb" in
```

Comment:
typically I do `__MODULE__` for the child name

Reply:
👍